### ISSUE TITLE
sureha depricated call async_add_job

### ISSUE THREAD

**Issue by ChristophCaina** [benleb/surepetcare#58]
the following warning about deprication is shown with HA 2026.6.0b0

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:184
First occurred: 8:57:30 PM (5 occurrences)
Last logged: 8:59:00 PM

Detected that custom integration 'sureha' calls `async_add_job`, which is deprecated and will be removed in Home Assistant 2025.4; Please review https://developers.home-assistant.io/blog/2024/03/13/deprecate_add_run_job for replacement options at custom_components/sureha/__init__.py, line 182: self.hass.async_add_job(, please create a bug report at https://github.com/benleb/sureha/issues
Detected that custom integration 'sureha' calls `async_add_job`, which is deprecated and will be removed in Home Assistant 2025.4; Please review https://developers.home-assistant.io/blog/2024/03/13/deprecate_add_run_job for replacement options at custom_components/sureha/__init__.py, line 188: self.hass.async_add_job(, please create a bug report at https://github.com/benleb/sureha/issues
Detected that custom integration 'sureha' calls `async_add_job`, which is deprecated and will be removed in Home Assistant 2025.4; Please review https://developers.home-assistant.io/blog/2024/03/13/deprecate_add_run_job for replacement options at custom_components/sureha/__init__.py, line 194: self.hass.async_add_job(, please create a bug report at https://github.com/benleb/sureha/issues
```

**Comment by pdwonline** [comment]
same here. Is this still maintained? (the fix seems to be ready as a PR)
Or do we need to fork this?

**Comment by ChristophCaina** [comment]
> same here. Is this still maintained? (the fix seems to be ready as a PR) Or do we need to fork this?

I'm not sure to which PR you are referring, but my last PR - which hasn't been merged for months now, will probably not fix this deprication warning.
In fact, I have fixed some other ones, that came with HA 2024.1.

I will have a look into it to see, if I can fix this issue, too...

**Comment by ChristophCaina** [comment]
should now be included in PR #59
